rareapi/fixtures/rareusers.json

```diff
@@ -20,5 +20,16 @@
           "active": true,
           "created_on": "2020-08-28T14:51:39.989Z"
       }
-  }
+  },
+  {
+    "model": "rareapi.rareusers",
+    "pk": 3,
+    "fields": {
+        "bio": "I am Mike",
+        "profile_image_url": "",
+        "user": 3,
+        "active": true,
+        "created_on": "2020-11-16 20:16:14.158580"
```

Comment:
I am getting some warnings when running these fixtures and I think it's because of the format of these date strings. If you remove the space in that string and replace it with `T` I think that will work correctly. You can check the `created_on` value in the one subscription object in the `subscriptions.json` fixture to see what I am talking about. I think this is something to do with timezones? Lol I actually don't really even know the difference between the two in practice but I am pretty sure that adding the `T` will make the warnings not happen when fixtures are run...